<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mplayer</class>
 <widget class="QMainWindow" name="mplayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mplayer</string>
  </property>
  <widget class="QWidget" name="movieForm">
   <widget class="QPushButton" name="addBtn">
    <property name="geometry">
     <rect>
      <x>459</x>
      <y>274</y>
      <width>60</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>添加</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delBtn">
    <property name="geometry">
     <rect>
      <x>528</x>
      <y>274</y>
      <width>60</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextBtn">
    <property name="geometry">
     <rect>
      <x>234</x>
      <y>353</y>
      <width>60</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>下一个</string>
    </property>
   </widget>
   <widget class="QPushButton" name="playBtn">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>353</y>
      <width>60</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>播放</string>
    </property>
   </widget>
   <widget class="QPushButton" name="preBtn">
    <property name="geometry">
     <rect>
      <x>162</x>
      <y>353</y>
      <width>60</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>上一个</string>
    </property>
   </widget>
   <widget class="QPushButton" name="speedBtn">
    <property name="geometry">
     <rect>
      <x>491</x>
      <y>369</y>
      <width>92</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>播放速度</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopBtn">
    <property name="geometry">
     <rect>
      <x>89</x>
      <y>353</y>
      <width>60</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>停止</string>
    </property>
   </widget>
   <widget class="QPushButton" name="timeBtn">
    <property name="geometry">
     <rect>
      <x>491</x>
      <y>341</y>
      <width>92</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>时间跳转</string>
    </property>
   </widget>
   <widget class="QSlider" name="currentlBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>323</y>
      <width>581</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="volumeSlider">
    <property name="geometry">
     <rect>
      <x>354</x>
      <y>357</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>60</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="timeLabel">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>310</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="view2">
    <property name="geometry">
     <rect>
      <x>312</x>
      <y>358</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>音量：</string>
    </property>
   </widget>
   <widget class="QListWidget" name="list">
    <property name="geometry">
     <rect>
      <x>459</x>
      <y>34</y>
      <width>130</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>17</y>
      <width>425</width>
      <height>300</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="view">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>11</y>
       <width>405</width>
       <height>280</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>495</x>
      <y>14</y>
      <width>62</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>播放列表</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
